src/org/labkey/test/params/experiment/MetricUnit.java

```diff
@@ -0,0 +1,7 @@
+package org.labkey.test.params.experiment;
+
+public interface MetricUnit<T extends MetricUnit<T>>
```

Comment:
This sort of self-referential type parameter is just a hack we do to allow chaining in some pages and components. It shouldn't be necessary here.
```suggestion
public interface MetricUnit
```

Reply:
Removed. I was trying to work around an issue and forgot I had added this.